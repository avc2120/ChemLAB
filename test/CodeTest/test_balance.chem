/* Balance Test */

function main()
{
	
	Balance("MgO, Fe == Fe2O3, Mg");

	Balance("Cu2S, O2 == Cu, SO2");

	Balance("Mg, HCl == MgCl2, H2");

	Balance("Ag, HNO3 == AgNO3, NO, H2O");

	Balance("Cl2, CaO2H2 == CaCl2O2, CaCl2, H2O");

	Balance("HNO3, Cu == CuN2O6, H2O, NO");

	Balance("C3H8O, O2 == CO2, H2O");

	Balance("KBr, KMnO4, H2SO4 == Br2, MnSO4, K2SO4, H2O");

	Balance("HNO3, Cu == CuN2O6, H2O, NO");

}