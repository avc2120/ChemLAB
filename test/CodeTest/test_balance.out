3 MgO 2 Fe --> 1 Fe2O3 3 Mg
1 Cu2S 1 O2 --> 2 Cu 1 SO2
1 Mg 2 HCl --> 1 MgCl2 1 H2
3 Ag 4 HNO3 --> 3 AgNO3 1 NO 2 H2O
2 Cl2 2 CaO2H2 --> 1 CaCl2O2 1 CaCl2 1 H2O
8 HNO3 3 Cu --> 3 CuN2O6 4 H2O 2 NO
2 C3H8O 9 O2 --> 6 CO2 8 H2O
10 KBr 2 KMnO4 8 H2SO4 --> 5 Br2 2 MnSO4 6 K2SO4 1 H2O
8 HNO3 3 Cu --> 3 CuN2O6 4 H2O 2 NO
